docs/docs-components/Overview.js

```diff
@@ -48,38 +53,19 @@ Not sure which component to use? [Set up time with the Gestalt team.](/get_start
             type="guidelines"
           />
         </IllustrationContainer>
-        <IllustrationContainer justifyContent="start">
-          <Flex gap={6} alignItems="center">
-            <Box aria-hidden>
-              <Text size="200">Sort by</Text>
-            </Box>
-            <RadioGroup
-              id="overview"
-              legendDisplay="hidden"
-              legend="Sort Gestalt components alphabetically or categorically"
-              direction="row"
-            >
-              <RadioGroup.RadioButton
-                checked={order === 'alphabetical'}
-                id="alphabetical"
-                label="Alphabetical"
-                name="overviewSort"
-                onChange={() => setOrder('alphabetical')}
-                value="alphabetical"
-                size="sm"
-              />
-              <RadioGroup.RadioButton
-                checked={order === 'category'}
-                id="category"
-                label="Category"
-                name="overviewSort"
-                onChange={() => setOrder('category')}
-                value="category"
-                size="sm"
-              />
-            </RadioGroup>
-          </Flex>
+
+        <IllustrationContainer>
+          <Box flex="none" minWidth={230} width="40%">
+            <SegmentedControl
+              items={sortOrders.map(capitalizeFirstLetter)}
```

Comment:
TBH I hadn't worked with SegmentedControl much before and the way it uses indices instead of values is a bit annoying